scripts/lib/CIME/XML/namelist_definition.py

```diff
@@ -445,4 +453,4 @@ def get_default_value(self, item, attribute=None):
             all_attributes.update(attribute)
 
         value = self.get_value_match(item.lower(), all_attributes, True)
-        return self._split_defaults_text(value)
+        return value
```

Comment:
The original motivation for this change was seeing that `self.get_value_match` can now return `None`; but as I looked closely, I saw that there was a double-call to `self._split_defaults_text(value)` (at the end of `self.get_value_match` and here) that looks like a bug, so I have removed that double-call. I don't see any calls to this `get_default_value` routine from within cime, but it's possible that components use it in their buildnml.